packages/expo-notifications/src/NotificationChannelGroupManager.web.ts

```diff
@@ -0,0 +1,6 @@
+import { NotificationChannelGroupManager } from './NotificationChannelGroupManager.types';
+
+export default {
+  addListener: () => {},
```

Comment:
Lately we've been flipping the "web" and native extensions so empty shims like this are the default and things like `.native`, `.web` are used when a bundler that supports platform extensions is in use. In the future any other platform someone attempts to use would default to this empty unimplemented module as the default. Related https://github.com/expo/expo/pull/8502